src/app/api.py

```diff
@@ -36,6 +36,7 @@
     title="Jira Bugzilla Integration (JBI)",
     description="JBI v2 Platform",
     version="3.0.2",
+    debug=True,
```

Comment:
Do we want to make this an env var?

Reply:
😱 we do, but that wasn't my intent for this PR. Good catch.

Reply:
Feel free to resolve this and we can make the changes later!